### PR TITLE
Fix/cli error msg

### PR DESCRIPTION
**Pull request by jbout**

### CODE REVIEW

**Comment by boajer** [comment]
Nice catch!

File: models/classes/routing/CliController.php [hunk 1]
```diff
@@ -23,7 +23,6 @@
 use oat\oatbox\action\ActionService;
 use oat\oatbox\action\ResolutionException;
 use common_report_Report as Report;
-use oat\oatbox\action\Help;
```
Comment (boajer) [inline models/classes/routing/CliController.php]: I have feeling  you've removed the only occurrence of  `oat\oatbox\action\Help`  usage across codebase.
https://github.com/search?l=PHP&p=7&q=org%3Aoat-sa+oat%5Coatbox%5Caction%5CHelp&type=Code

Should be consider risky step and remove class itself?  @hutnikau @jbout

Comment (bugalot) [reply models/classes/routing/CliController.php]: It seems to me also it's the only one in the code base. In favour to remove the class.

Comment (jbout) [reply models/classes/routing/CliController.php]: Agreed, but it is in generis

**Comment by boajer** [comment]
Merged, thank you all.

**Pull request merged**